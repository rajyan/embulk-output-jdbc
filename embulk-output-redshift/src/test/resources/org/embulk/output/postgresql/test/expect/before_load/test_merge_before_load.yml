table: test1
mode: merge
before_load: "delete from test1 where int_item = 9"
