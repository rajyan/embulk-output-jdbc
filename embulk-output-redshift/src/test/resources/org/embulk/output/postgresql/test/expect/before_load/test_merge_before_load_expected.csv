A001,9,a
A002,0,b
A003,9,c
B001,0,z
